### DISCUSSION TITLE
Indeo video codec support in cx23.7.1?

### DISCUSSION THREAD

**Discussion by yochenhsieh** [Sikarugir-App/Sikarugir#48] [General]
I tried to run Star Trek The Next Generation Klingon Honor Guard using Kegworks. With nglide installed, the game starts. But in-game video is black.
This game needs indeo codec to play video, but it does not work with the dll installed by winetricks. I see ir50_32.dll is implemented in wine 9.0/crossover 24. Is it possible to backport this dll to current cx23.7.1 engine?
Thank you

**Comment by pythonismyfavorite** [comment]
idk

**Comment by Gcenx** [comment]
I’ve back-ported `ir50_32` into the unpublished `WS12WineCX23.7.1_2`, that also allows more OpenGL games to run and will include the Battle.net fixes from CX24.0.7 (once the sources are pushed)

**Reply by Gcenx** [reply]
@yochenhsieh `WS12WineCX23.7.1_2` has now been published

**Reply by yochenhsieh** [reply]
[LastRunWine.log](https://github.com/user-attachments/files/18581791/LastRunWine.log)
Have this error
(wine:2245): GStreamer-Video-CRITICAL **: 10:23:48.950: gst_video_info_from_caps: assertion 'gst_caps_is_fixed (caps)' failed
00c4:fixme:quartz:amt_to_wg_format_video Unknown format type {00000000-0000-0000-0000-000000000000}.

Seems like this bug: (no video but sound plays)
https://bugs.winehq.org/show_bug.cgi?id=56936
But I don't have 'media_source_get_service_GetService Unsupported service' message.

**Reply by Gcenx** [reply]
Checking over the bug and the other linked bug this hasn’t been resolved upstream yet so there’s nothing else I can do
